src/main/java/frc/robot/commands/StartShooterCommandGroup.java

```diff
@@ -1,37 +1,20 @@
 package frc.robot.commands;
 
-import edu.wpi.first.wpilibj2.command.CommandGroupBase;
 import frc.robot.subsystems.BallShooterSubsystem;
 import frc.robot.subsystems.BallFeederSubsystem;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 
-public class StartShooterCommandGroup extends CommandGroupBase {
 
-    public StartShooterCommandGroup(BallShooterSubsystem subsystem, BallFeederSubsystem subsystem)
+public class StartShooterCommandGroup extends SequentialCommandGroup
+{
+    public StartShooterCommandGroup(BallShooterSubsystem ballShooter, BallFeederSubsystem ballFeeder)
     {
-        addSequential(new RunBallFeeder());
-        addSequential(new RunBallShooter());
-
+        // new SequentialCommandGroup(new RunBallFeeder(ballFeeder, 0.5), new WaitCommand(1), new RunBallShooter(ballShooter, 0.5, 0.7));
+        addCommands(
+            new RunBallShooter(ballShooter, 0.2, 0.6),
+            new WaitCommand(1.0),
+            new RunBallFeeder(ballFeeder, 0.2)
+        );
     }
```

Comment:
Your parenthesis got divorced? Why are they on two separate planes of existence? Do they hate each other? Confused.